### ISSUE TITLE
[FEAT]: Better Edit Card Modal

### ISSUE THREAD

**Issue by 4Source** [st3v3nmw/obsidian-spaced-repetition#988]
### Is your feature request related to a problem? Please describe.

Currently the Edit Card Modal is really bad to read or edit. 

### Describe the solution you'd like

Split up front and back of flashcards in different text boxes

**Comment by kocoai** [comment]
I agree, I think the modal in the current state should be disabled completely (open in new tab only) until we have a better modal... The modal is completely buggy in mobile: impossible to close => need to force quit app
